shopfloor_reception_mobile/static/src/scenario/reception.js

```diff
@@ -367,6 +367,9 @@ const Reception = {
                             path: "quantity_done",
                             label: "Qty done",
                             display_no_value: true,
+                            renderer: (rec, field) => {
+                                return rec.quantity_done + rec.backorders_quantity_done;
+                            },
```

Comment:
this shouldn't be the frontend responsibility to compute that.

Reply:
Hmmmm I think it's important to make the distinction between the qty done in the picking and the qty done from the backorders, isn't it?
Because what if auto_post is not enabled, we start working on a picking, and then we enable it and keep working on it, we may not have accurate qties done.

To me the simplest is to give the frontend both pieces of information and let it decide what to do with it.